src/StrokeEngine.cpp

```diff
@@ -746,86 +247,20 @@ void StrokeEngine::_stroking() {
         }
         
         // Delay 10ms 
-        vTaskDelay(10 / portTICK_PERIOD_MS);
-    }
-}
-
-void StrokeEngine::_streaming() {
-
-    while(1) { // infinite loop
-
-        // Suspend task, if not in STREAMING state
-        if (_state != STREAMING) {
-            vTaskSuspend(_taskStreamingHandle);
-        }
-        
-        // Delay 10ms 
-        vTaskDelay(10 / portTICK_PERIOD_MS);
-    }
-}
-
-void StrokeEngine::_applyMotionProfile(motionParameter* motion) {

```

Comment:
I do understand why you removed this code. But this code contains fundamentally important safeguards. We must ensure that these safeguards stay active. Probably the best place would be motor.hpp.